src/qililab/qprogram/crosstalk_matrix.py

```diff
@@ -14,9 +14,51 @@
 
 from typing import Collection
 
+import numpy as np
+
 from qililab.yaml import yaml
 
 
+class FluxVector:
```

Comment:
Pretty much, for the time being this is only used for the xtalk. But it's better to have an object for the `__matmul__` method rather than just a typed dictionary, this was something we agreed with @fedonman (correct me if wrong)

Reply:
Yeah, I agree with the objectification.